plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergProjectionPushdownPlans.java

```diff
@@ -129,7 +129,7 @@ public void testDereferencePushdown()
 
         getQueryRunner().execute(format(
                 "CREATE TABLE %s (col0, col1) WITH (partitioning = ARRAY['col1']) AS" +
-                        " SELECT CAST(row(5, 6) AS row(x bigint, y bigint)) AS col0, 5 AS col1 WHERE false",
```

Comment:
why this change?

Reply:
There's an implicit change here that we can now enforce all predicates on empty tables. That broke this test because the table is empty but was not expecting a predicate to be enforced.

Reply:
But doesn't it mean that now iceberg will fail on valid sql ?

Reply:
Spoke with Konrad offline about this, not a problem